main.rb

```diff
@@ -0,0 +1,70 @@
+require_relative 'app'
+
+def list_all_options
+  puts 'Please choose an option by entering a number:'
```

Comment:
- [ ] Kindly make sure to fix this issue. when i first run the app the option does not show for me to choose but rather enter my Age.

<img width="1011" alt="Screenshot 2023-06-09 at 06 48 25" src="https://github.com/FrankMugagga/OOP-school-library/assets/81444338/a9f4755d-fbb4-4b62-8754-26a7dea353ce">

- [ ] Check out this example
<img width="582" alt="Screenshot 2023-06-09 at 07 02 19" src="https://github.com/FrankMugagga/OOP-school-library/assets/81444338/01b18456-d7ab-4d93-a285-82c468ab393f">